packages/orchestration/src/orchestration-client.ts

```diff
@@ -78,31 +202,73 @@ export function constructCompletionPostRequestFromJson(
   }
 }
 
+function addStreamOptions(
+  moduleConfigs: ModuleConfigs,
+  streamOptions?: StreamOptions
+): OrchestrationConfig {
+  const llm = streamOptions?.llm ?? {};
+  const outputFiltering = streamOptions?.outputFiltering ?? {};
+  const chunkSize = streamOptions?.chunkSize;
+
+  return {
+    stream: true,
+    stream_options: {
+      chunk_size: chunkSize
+    },
+    module_configurations: {
+      ...moduleConfigs,
+      llm_module_config: {
+        ...moduleConfigs.llm_module_config,
+        stream_options: {
+          include_usage: llm.includeUsage ?? true,
+          ...llm
+        }
+      },
+      ...(Object.keys(outputFiltering).length && {
+        filtering_module_config: {
+          ...moduleConfigs.filtering_module_config,
+          output: {
+            ...(moduleConfigs.filtering_module_config?.output || {}),
+            stream_options: outputFiltering
+          }
+        }
+      })
+    }
+  } as OrchestrationConfig; // TODO: Remove typecast when types are re-generated;
```

Comment:
[q] Can we fix this within this PR? I see some type issues if not casting it.

Reply:
[req] I think you probably need to add a type check for output as well so that filters does not evaluate to undefined. 

```
...(moduleConfigs.filtering_module_config?.output && {
            output: {
            filters: moduleConfigs.filtering_module_config.output.filters,
            stream_options: outputFiltering
          }})
```

I am guessing there is a better way to write this.